test/Microsoft.SqlTools.ServiceLayer.Test/LanguageServer/PeekDefinitionTests.cs

```diff
@@ -251,17 +267,17 @@ public void GetFullObjectNameFromQuickInfoWithNullStringsTest()
 
             string objectName = null;
             string quickInfoText = "table master.dbo.testTable";
-            string result = peekDefinition.GetFullObjectNameFromQuickInfo(quickInfoText, objectName);
+            string result = peekDefinition.GetFullObjectNameFromQuickInfo(quickInfoText, objectName, StringComparison.Ordinal);
```

Comment:
I added a separate test for OrdinalIgnoreCase. So I'm leaving this as Ordinal which was the original test intent.